android/src/main/java/com/google/samples/apps/iosched/service/SessionCalendarService.java

```diff
@@ -102,9 +102,7 @@ protected void onHandleIntent(Intent intent) {
                         processAllSessionsCalendar(resolver, getCalendarId(intent)));
                 sendBroadcast(new Intent(
                         SessionCalendarService.ACTION_UPDATE_ALL_SESSIONS_CALENDAR_COMPLETED));
-            } catch (RemoteException e) {
-                LOGE(TAG, "Error adding all sessions to Google Calendar", e);
-            } catch (OperationApplicationException e) {
+            } catch (Exception e) {
```

Comment:
IMO Your approach has a better readability, but it requires higher version of Java (which is not a big problem now, actually). For those who still need to work with Java 6, keeping the old lines is better. And generally, catching a Abstract Exception is not a good practice.